src/main/java/net/dv8tion/jda/api/Permission.java

```diff
@@ -223,7 +223,7 @@ public static Permission getFromOffset(int offset)
 
     /**
      * A set of all {@link net.dv8tion.jda.api.Permission Permissions} that are specified by this raw long representation of
-     * permissions. The is best used with the getRaw methods in {@link net.dv8tion.jda.api.entities.Role Role} or
+     * permissions. This is best used with the getRaw methods in {@link net.dv8tion.jda.api.entities.Role Role} or
```

Comment:
I wasn't sure if this was a typo (Probably was?) so I corrected this...
Or should it be `The method is ...`?